test/unit/test_compute_chunk_size.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+import pytest
+
+try:
+    from snowflake.connector.constants import (
+        S3_MAX_OBJECT_SIZE,
+        S3_MAX_PART_SIZE,
+        S3_MIN_PART_SIZE,
+    )
+    from snowflake.connector.file_transfer_agent import _chunk_size_calculator
+except ImportError:
```

Comment:
Why do need this try catch block?

Reply:
old driver doesn't have `file_transfer_agent` module